packages/atomic/src/components/facets-v1/atomic-facet/atomic-facet.tsx

```diff
@@ -115,6 +124,17 @@ export class AtomicFacet implements InitializableComponent {
     };
   }
 
+  public componentShouldUpdate(next: unknown, prev: unknown, propName: string) {
+    if (propName === 'facetState') {
```

Comment:
Is the `facetState` prop typed in any manner ?

What would need to be changed for this to start failing silently on us ? 
If someone rename the prop ?